src/staging/validation.py

```diff
@@ -50,9 +49,7 @@ def get_masterlist(postcode_masterlist) -> pd.Series:
 
 @time_logger_wrap
 @exception_wrap
-def validate_post_col(
-    df: pd.DataFrame, postcode_masterlist: str
-) -> bool:
+def validate_post_col(df: pd.DataFrame, postcode_masterlist: str) -> bool:
```

Comment:
add postcode_masterlist arg

Reply:
Add returns to doc string - invalid_df, unreal_df

Reply:
Could add comment to make distiction between invalid and unreal - what distinguishes each